### ISSUE TITLE
Copy/paste error: Conversion BCP47 to SAP1

### ISSUE THREAD

**Issue by albertmink** [abapGit/abapGit#6970]
https://github.com/abapGit/abapGit/blob/8b2ab1e5d9aa93bf42fe0eb68f9c7a8cf25425ba/src/utils/zcl_abapgit_convert.clas.abap#L235

**Comment by albertmink** [comment]
An no exemptions in here I assume 
https://github.com/abapGit/abapGit/blob/8b2ab1e5d9aa93bf42fe0eb68f9c7a8cf25425ba/src/utils/zcl_abapgit_convert.clas.abap#L247-L254
Why can't we just call?
```ABAP
            data(conv) = cl_aff_language_converter=>create_instance( ).
            conv->bcp47_to_sap1( im_lang_bcp47 ).
```

**Comment by mbtools** [comment]
Closed by #6971
